scripts/dokka/README.md

```diff
@@ -0,0 +1,150 @@
+### Description
```

Comment:
I would rename the folder into something like `scripts/publish`. Here, we'd want to emphasise the functionality, not the tool.